
Cтрока 1 (04.06.2022 14:08:31)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26

Cтрока 2 (04.06.2022 14:08:38)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26

Cтрока 3 (04.06.2022 14:08:38)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26

Cтрока 1 (04.06.2022 14:33:42)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26

Cтрока 2 (04.06.2022 14:33:54)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26

Cтрока 3 (04.06.2022 14:33:54)
System.Data.DataException: При инициализации базы данных возникло исключение. Дополнительные сведения можно получить через объект InnerException. ---> System.Data.Entity.Core.EntityException: Произошла ошибка базового поставщика в Open. ---> System.Data.SqlClient.SqlException: Cannot attach the file 'C:\Users\User\Source\Repos\SBC\App_Data\SBC.mdf' as database 'SBC'.
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   в System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   в System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   в System.Data.SqlClient.SqlConnection.Open()
   в System.Data.Entity.Infrastructure.Interception.InternalDispatcher`1.Dispatch[TTarget,TInterceptionContext](TTarget target, Action`2 operation, TInterceptionContext interceptionContext, Action`3 executing, Action`3 executed)
   в System.Data.Entity.Infrastructure.Interception.DbConnectionDispatcher.Open(DbConnection connection, DbInterceptionContext interceptionContext)
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<>c__DisplayClass1.<Execute>b__0()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Core.EntityClient.EntityConnection.Open()
   в System.Data.Entity.Core.Objects.ObjectContext.EnsureConnection(Boolean shouldMonitorTransactions)
   в System.Data.Entity.Core.Objects.ObjectContext.ExecuteInTransaction[T](Func`1 func, IDbExecutionStrategy executionStrategy, Boolean startLocalTransaction, Boolean releaseConnectionOnSuccess)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<>c__DisplayClass7.<GetResults>b__5()
   в System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.Execute[TResult](Func`1 operation)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.GetResults(Nullable`1 forMergeOption)
   в System.Data.Entity.Core.Objects.ObjectQuery`1.<System.Collections.Generic.IEnumerable<T>.GetEnumerator>b__0()
   в System.Data.Entity.Internal.LazyEnumerator`1.MoveNext()
   в System.Linq.Enumerable.FirstOrDefault[TSource](IEnumerable`1 source)
   в System.Data.Entity.Internal.EdmMetadataRepository.QueryForModelHash(Func`2 createContext)
   в System.Data.Entity.Internal.ModelCompatibilityChecker.CompatibleWithModel(InternalContext internalContext, ModelHashCalculator modelHashCalculator, Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.Internal.InternalContext.CompatibleWithModel(Boolean throwIfNoMetadata, DatabaseExistenceState existenceState)
   в System.Data.Entity.CreateDatabaseIfNotExists`1.InitializeDatabase(TContext context)
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   --- Конец трассировки внутреннего стека исключений ---
   в System.Data.Entity.Internal.InternalContext.PerformInitializationAction(Action action)
   в System.Data.Entity.Internal.InternalContext.PerformDatabaseInitialization()
   в System.Data.Entity.Internal.RetryAction`1.PerformAction(TInput input)
   в System.Data.Entity.Internal.LazyInternalContext.InitializeDatabaseAction(Action`1 action)
   в System.Data.Entity.Internal.InternalContext.GetEntitySetAndBaseTypeForType(Type entityType)
   в System.Data.Entity.Internal.Linq.InternalSet`1.Initialize()
   в System.Data.Entity.Internal.Linq.InternalSet`1.get_InternalContext()
   в System.Data.Entity.Infrastructure.DbQuery`1.System.Linq.IQueryable.get_Provider()
   в System.Linq.Queryable.Any[TSource](IQueryable`1 source)
   в SBC.Entities.ApplicationDbContext.Populate() в C:\Users\User\Source\Repos\SBC\Entities\ApplicationDbContext.cs:строка 26
